lib/EnsEMBL/REST/Model/Variation.pm

```diff
@@ -330,7 +330,7 @@ sub pops_as_hash {
   my $population;
   $population->{frequency} = 0 + $allele->frequency(); # Add 0 to treat it as numeric (to avoid quoting)
   $population->{population} = $allele->population->name();
-  $population->{allele_count} = 0 + $allele->count(); # Add 0 to treat it as numeric (to avoid quoting)
+  $population->{allele_count} = 0 + $allele->count() if defined $allele->count(); # Add 0 to treat it as numeric (to avoid quoting)
```

Comment:
I would really recommend using a block rather than postfix form of if here. It might be slightly more verbose but at least it makes it one hundred percent clear that it is the whole assignment to $population->{allele_count} that is controlled by the condition.

Reply:
Modified to use block if